core/server/data/migrations/versions/4.1/01-fix-backup-content-permission-typo.js

```diff
@@ -0,0 +1,15 @@
+const logging = require('../../../../../shared/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(async function up(knex) {
+    const typoedPermisson = await knex.select('*').from('permissions').where('action_type', 'backupContect').first();
```

Comment:
You typoed `Permission` in `typoedPermisson` 😝

Reply:
:joy: fucks sake